compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.td

```diff
@@ -8,10 +8,182 @@
 #define IREE_CODEGEN_DIALECT_IREEGPUOPS
 
 include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUDialect.td"
+include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUInterfaces.td"
+include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td"
 include "mlir/Interfaces/SideEffectInterfaces.td"
 include "mlir/IR/OpAsmInterface.td"
 include "mlir/IR/OpBase.td"
 
+//===----------------------------------------------------------------------===//
+// MultiMmaOp
+//===----------------------------------------------------------------------===//
+
+def IREEGPU_MultiMmaOp : Op<IREEGPU_Dialect, "multi_mma", [
+    Pure,
+    AllTypesMatch<["acc", "result"]>,
+    ]> {
+  let summary = "Models a contraction of multiple mma operations";
+  let description = [{
+    Computes the sum of inner MMA operations along a set of outer dimensions.
+    Logically matches closely with a `vector.contraction` operation, however
+    the combiner type is a specific intrinsic rather than a generic combiner
+    type.
```

Comment:
It would probably be good to plumb an upstream attribute like this upstream through vector.contract also.